microapps/ScalebarApp/app/build.gradle.kts

```
@@ -0,0 +1,94 @@
+/*
+ *
+ *  Copyright 2024 Esri
```

```suggestion
 *  Copyright 2025 Esri
```
This needs to be updated in all the files. I will only comment here.